src/test/resources/com/cloudbees/plugins/credentials/impl/validCerts.pem

```diff
@@ -0,0 +1,20 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
is there something wrong with the [existing](https://github.com/jenkinsci/credentials-plugin/tree/master/src/test/resources/com/cloudbees/plugins/credentials/impl) PEM files that means you can not use them?

Reply:
The existing PEM files cannot be used in FIPS mode because the encryption algorithm used is not FIPS compliant.
Error:
org.bouncycastle.crypto.fips.FipsUnapprovedOperationError: salt must be at least 128 bits